### PR TITLE
Updates to case sensitivity and "nag" captiveOption

### PR DESCRIPTION
**Pull request by ReckerXF**
I have made it so the "nag" captiveOption has a sub-option for where to display the text: Center or Top. With this, I have centered the drawText a bit more. I have also made it so all captiveOptions are now non-case sensitive :)

**Edit:** I've also included an instructional message to teach users how to go and add the API ID. Lots of people were confused in my server on how to do it and were asking staff. This will help them along :)

_I did not edit the version number as I am not sure what you have planned for versions._

### CODE REVIEW

**Comment by ReckerXF** [comment]
Sorry about the mistake with one of the commits. If you need to read the file to see the changes better, here ya go: https://github.com/Blizzard098/sonoran_forcereg/blob/master/forcereg/cl_forcereg.lua

File: forcereg/CHANGEMEconfig_forcereg.lua [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ]]
 local config = {
-    enabled = false,
+    enabled = true,
```
Comment (SonoranBrian) [inline forcereg/CHANGEMEconfig_forcereg.lua]: Default plugin config should show enabled as false

Comment (nathanctech) [reply forcereg/CHANGEMEconfig_forcereg.lua]: Agreed, default config needs to be disabled.

File: forcereg/CHANGEMEconfig_forcereg.lua [hunk 2]
```diff
@@ -20,15 +20,21 @@ local config = {
         Freeze: Freezes the player at their spawn point with a big notification.
         Whitelist: Prevents connection to the server entirely via deferrals (WARNING: NOT COMPATIBLE WITH ADAPTIVE CARD RESOURCES)
     ]]
-    captiveOption = "nag",
+    captiveOption = "Nag",
```
Comment (nathanctech) [inline forcereg/CHANGEMEconfig_forcereg.lua:23]: Not strictly needed, lua is case insensitive despite the comment above being uppercased. Seems fine, though.

File: forcereg/cl_forcereg.lua [hunk 3]
```diff
@@ -23,11 +23,11 @@ if pluginConfig.enabled then
             Wait(1)
             id = identifier
             print(("Identifier %s does not exist."):format(identifier))
-            if pluginConfig.captiveOption == "nag" then
+            if pluginConfig.captiveOption:lower() == "nag" then
```
Comment (nathanctech) [inline forcereg/cl_forcereg.lua]: Not needed if the above config is left lowercase. Is there a reason for this since adding `lower` just adds complexity?

**Review by nathanctech — CHANGES_REQUESTED** [review]

File: forcereg/cl_forcereg.lua (hunk 3 above)
Comment (ReckerXF) [reply forcereg/cl_forcereg.lua]: Yeah sorry I thought Lua was case sensitive with strings. My bad.

File: forcereg/CHANGEMEconfig_forcereg.lua (hunk 1 above)
Comment (ReckerXF) [reply forcereg/CHANGEMEconfig_forcereg.lua]: Yeah sorry forgot to set it back to false.

File: forcereg/CHANGEMEconfig_forcereg.lua (hunk 2 above)
Comment (ReckerXF) [reply forcereg/CHANGEMEconfig_forcereg.lua:23]: See other comment.

**Comment by ReckerXF** [comment]
Made requested edits. My apologies again, thought Lua was case sensitive.

**Review by nathanctech — APPROVED** [review]

**Pull request merged**